iroh-metrics/Cargo.toml

```diff
@@ -15,12 +15,16 @@ rust-version = "1.72"
 workspace = true
 
 [dependencies]
-prometheus-client = { version = "0.22.0", optional = true }
-once_cell = "1.17.0"
-tracing = "0.1"
-hyper = { version = "0.14.25", features = ["server", "client", "http1", "tcp"] }
+anyhow = "1.0.75"
 erased_set = "0.7"
+http-body-util = "0.1.0"
+hyper = { version = "1", features = ["server", "http1"] }
+hyper-util = { version = "0.1.1", features = ["tokio"] }
+once_cell = "1.17.0"
+prometheus-client = { version = "0.22.0", optional = true }
 struct_iterable = "0.1"
+tokio = "1"
```

Comment:
this pull in all features, please minimize those as needed

Reply:
ccb5c50f